src/latest/lib/bindings.sc

```diff
@@ -75,11 +76,10 @@ class MarathonStorage(args: List[String] = helpers.InternalHelpers.argsFromEnv)
   implicit lazy val client = storage.client
   implicit lazy val underlyingModule = StorageModule(storage, "mesos-bridge-name")
   lazy val store = {
-    val s = storage.store
-    // We need to call this method before using the storage module if it is defined
-    s.getClass.getMethods.find(_.getName == "markOpen").foreach { m =>
```

Comment:
I think, `markOpen` wasn't there since 1.6.0, so it makes sense to keep it conditional, I think.

Reply:
It was in 1.6.0

```
23: 35 $ git show v1.6.0:src/main/scala/mesosphere/marathon/util/Openable.scala
package mesosphere.marathon
package util
...
```

Reply:
Thanks!